quelpa.el

```diff
@@ -210,10 +218,11 @@ On error return nil."
                               (package-buffer-info))
                      (`tar (insert-file-contents-literally file)
                            (tar-mode)
-                           (if (help-function-arglist 'package-tar-file-info)
-                               ;; legacy `package-tar-file-info' requires an arg
-                               (package-tar-file-info file)
-                             (with-no-warnings (package-tar-file-info)))))))))
+                           (with-no-warnings 
```

Comment:
This is just to prevent byte-compiler warnings, right?

Reply:
Yes, to prevent only version of `package-tar-file-info` is taking an argument